charts/prod-ohstaff-values.yaml

```diff
@@ -0,0 +1,116 @@
+# Values for oh-staff.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: uclalibrary/oral-history-staff-ui
+  tag: v0.0.1
+  pullPolicy: Always
+
+nameOverride: ""
+
+fullnameOverride: ""
+
+service:
```

Comment:
Setting the Service type to `NodePort` is something I had been doing for a while. Recently I realized this is unnecessary for how we are funneling traffic into the cluster. I'll explain this more next time we meet. 

Let's use the `ClusterIP` Service type for this app (and most likely all others moving forward). You would just change this stanza to the following:
```
service:
  type: ClusterIP
```

Reply:
Yes - it should be changed in the default `values.yaml` file as well.

Reply:
Changed  both files.